src/abstract.js

```diff
@@ -793,16 +793,93 @@ Sk.abstr.sattr = function (obj, nameJS, data, canSuspend) {
 };
 goog.exportSymbol("Sk.abstr.sattr", Sk.abstr.sattr);
 
-Sk.abstr.iter = function (obj) {
+
+Sk.abstr.iternext = function (it, canSuspend) {
+    return it.tp$iternext(canSuspend);
+};
+goog.exportSymbol("Sk.abstr.iternext", Sk.abstr.iternext);
+
+
+/**
+ * @constructor
+ */
+function seqIter(obj) {
```

Comment:
@albertjan ```seqIter``` is a constructor for an iteration object.  Such objects are only ever created and returned by ```Sk.abstr.iter```.  I think you are right that this shouldn't be global, but I'm not sure if we have an established convention for where to place such a constructor.